daemon/graphdriver/btrfs/dummy_unsupported.go

```diff
@@ -1,3 +1,3 @@
-// +build !linux !cgo
+// +build !linux !cgo !windows
```

Comment:
Because linux and windows are mutually exclusive:

```
(!linux OR !cgo OR !windows) == !(linux AND cgo AND windows) == true
```

Are you sure we need to change anything here? What the original buildtag meant: if it's not both linux and cgo, then btrfs is not supported. That should stay the same imho. Ping @tianon